client/src/components/FAB.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import styled from "@emotion/styled";
+import addSrc from "../assets/addCircle.svg";
+import PropTypes from "prop-types";
+
+function FAB({ displayForm }) {
```

Comment:
FAB is ninja-code to me 🔢 I would write the long version 😉 
I would write it in a arrowFunction with const FloatingAddButton, because now it seems like a function that creates the button. So the function should have a name like "createButton", etc.

Reply:
Good point. Will change it :)